strawberryfields/apps/data/sample.py

```diff
@@ -372,3 +372,111 @@ class Formic(MoleculeDataset):
     n_mean = 1.56
     threshold = False
     T = 0
+
+
+class Water(SampleDataset):
+    """Vibrational dynamics of the water molecule.
+
+    The molecular parameters are obtained from Ref. :cite:`sparrow2018simulating`.
+
+    **Molecule:**
+
+    .. |water| image:: ../../../_static/water.png
+        :align: middle
+        :width: 200px
+        :target: javascript:void(0);
+
+    |water|
+
+    Attributes:
+        n_mean = 1/3
+        threshold = False
+        n_samples = 135000

```

Comment:
I'm getting 5k samples :thinking:

Reply:
So it's 5k samples for each time, and there are 27 times, thus 27*5000 = 135000. Will update the docstring to reflect this